packages/schematics/src/command-line/workspace-schematic.ts

```diff
@@ -146,7 +227,11 @@ async function executeSchematic(
     }
   });
   delete options._;
-  const logger = createConsoleLogger(true, process.stdout, process.stderr);
+
+  // Add support for interactive prompts
+  if (!options.noInteractive && options.interactive !== false) {
```

Comment:
```suggestion
  if (options.interactive) {
```

I think this should suffice after altering `parseOptions`